### ISSUE TITLE
about `w2 = 12.9`

### ISSUE THREAD

**Issue by tokoharu** [wind23/whr_renju#4]
In WHR, the value of parameter `w2` has a significant impact.
In `computing_rating.py`, the value is set to 'w2=12.9'.

Why is this value used? I would like to know if there is any evidence for this.

**Comment by wind23** [comment]
The parameter `w2` was tuned with the file `evaluate.py`, based on the database `renjunet_v10_20191120.rif`.

The database was randomly splitted, 90% was used as the training dataset, and 10% was used as the validation dataset. I tried different value of `w2`, and got the minimum loss with the value `12.9`.

**Comment by tokoharu** [comment]
Thank you very much! I understand your approach.

Then, did you also tune parameter `virtual_games`? ( for example, bad loss occur when `virtual_games=1` ? )

**Comment by wind23** [comment]
> Thank you very much! I understand your approach.
> 
> Then, did you also tune parameter `virtual_games`? ( for example, bad loss occur when `virtual_games=1` ? )

You are right, the loss of `virtual_games=1` was not as good as `virtual_games=2`.

**Comment by tokoharu** [comment]
Thank you !!!
My question was resolved, so I close this issue.
